src/components/application_manager/rpc_plugins/vehicle_info_plugin/include/vehicle_info_plugin/vehicle_info_plugin.h

```diff
@@ -71,6 +71,9 @@ class VehicleInfoPlugin : public plugins::RPCPlugin {
 
  private:
   void DeleteSubscriptions(app_mngr::ApplicationSharedPtr app);
+  void SelectVehicleDataForUnsubscribe(
+      const app_mngr::ApplicationSharedPtr& app,
+      std::set<mobile_apis::VehicleDataType::eType>& vehicle_data_for_unscribe);
```

Comment:
@IGapchuk don't you have alias `VehicleInfoSubscriptions` ?

Reply:
Fixed in 73c67bf4f764f384b64cc45f9a7a3087eff31f1f.